### PR TITLE
create 'cl config', default worksheets, some fixes

### PR DESCRIPTION
**Pull request by percyliang**
@kashizui

### CODE REVIEW

**Comment by kashizui** [comment]
Would you like to resolve the conflicts first or have me review it first?

**Comment by percyliang** [comment]
Let me fix it up a bit first.

**Comment by percyliang** [comment]
@kashizui by conflicts, you mean broken tests?

**Comment by kashizui** [comment]
I meant the merge conflicts, but I didn't notice the tests were broken too!

**Comment by percyliang** [comment]
You mean with your REST changes?

**Comment by kashizui** [comment]
My latest REST changes haven't been merged yet so I hope that isn't causing trouble, I'm just seeing on GitHub that "This branch has conflicts that must be resolved"?

**Comment by percyliang** [comment]
Ah, I see...strange, I see the message on GitHub I don't see any merge conflicts at the command-line with git.

**Comment by percyliang** [comment]
@kashizui tests pass, please take a look, along with https://github.com/codalab/codalab-worksheets/pull/197

**Comment by kashizui** [comment]
OK I will do so some time this week!

File: worker/main.py [hunk 1]
```diff
@@ -22,10 +22,10 @@
     parser.add_argument('--tag',
                         help='Tag that allows for scheduling runs on specific '
                              'workers.')
-    parser.add_argument('--bundle-service-url', required=True,
-                        help='URL of the bundle service, in the format '
-                             '<http|https>://<hostname>[:<port>]')
-    parser.add_argument('--work-dir', default='scratch',
+    parser.add_argument('--server', required=True,
```
Comment (kashizui) [inline worker/main.py:25]: Do we need to update our deployment scripts as well?

**Comment by kashizui** [comment]
LGTM, we just need to double check deployment will still work after this.

File: worker/main.py (hunk 1 above)
Comment (percyliang) [inline worker/main.py:25]: Yes, that has already been done.

**Pull request merged**